cmd/security-file-token-provider/res/docker/configuration.toml

```diff
@@ -0,0 +1,19 @@
+[SecretService]
+Scheme = "https"
+Server = "edgex-vault"
+ServerName = "edgex-vault"
```

Comment:
@bnevis-i I notice `ServerName` is missing from the non-docker `configuration.toml`. Is that an oversight?

Reply:
In fact, I forgot to git add that change.  Fixed.